features/205.CustonIoTAgent.feature

```diff
@@ -3,8 +3,8 @@ Feature: Test tutorial 205.IoT Agent Custom (XML)
     url: https://fiware-tutorials.readthedocs.io/en/latest/custom-iot-agent.html
     git-clone: https://github.com/FIWARE/tutorials.Custom-IoT-Agent.git
     git-directory: /tmp/tutorials.Custom-IoT-Agent
-    shell-commands: ./services create; ./services start
-    clean-shell-commands: ./services stop
+    shell-commands: docker rm fiware-iot-agent ; ./services create; ./services start
```

Comment:
Idem

Reply:
Beause docker is not removed. I must ensure that it is not there or the execution could  fail.

Reply:
if it is the case and it is no indicated in the tutorial, then there is a bug in the services script. Keep in mind.

NTC more